python/bridgestan/model.py

```diff
@@ -476,10 +503,7 @@ def param_unconstrain_json(
         dims = self.param_unc_num()
         if out is None:
             out = np.zeros(shape=dims)
-        elif out.size != dims:
```

Comment:
Thanks for removing---this looks like it can't fail given the logic and that we've already checked it if it exists.